app/templates/books/books.html

```diff
@@ -0,0 +1,38 @@
+{% extends "books/bookbase.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block title %}Ksiazki{% endblock %}
+{% block navbar %}
+    {{ super() }}
+{% endblock %}
+{% block page_content %}
+<div class="page-header">
+    <h1>Ksiazki</h1>
+        <table>
+            <tr>
+                <th>ID</th>
+                <th>Nazwa</th>
+                <th>Strony</th>
+            </tr>
+            {% for ksiazka in ksiazki %}
```

Comment:
W pliku app/templates/books/announcement.html mamy for announcement in announcements. Jak tam po angielsku to tu też
```suggestion
            {% for book in books %}
```